core-models/src/main/scala/com/mesosphere/usi/core/models/ReservationRecord.scala

```diff
@@ -0,0 +1,13 @@
+package com.mesosphere.usi.core.models
+
+/**
+  * * ReservationRecord is an immutable event of created reservations .
```

Comment:
Not sure if this really explains a reservation. Should we link to the reservation definition?

Nit: One `*` to many 🙂